src/app/modules/shared-components/components/input/input.component.html

```diff
@@ -5,6 +5,11 @@
     [ngClass]="{'has-button': buttonIcon}"
     [formControlName]="name"
   >
+  <span *ngIf="hasClearButton" class="clear-button">
+    <button (click)="clearInput()">
```

Comment:
Couldn't you just put a template variable on the input and do `#input.value=''` here?

Reply:
Done

Reply:
My remark was for `(click)="clearInput()"`

Reply:
After testing, the `#input.value=''` doesn't feat our needs, I've reverted to what was done previously.

As it is not the cleanest way to do it, in the future we should look into alternatives to see if there is a better way to do things, like using [this](https://angular.io/api/forms/DefaultValueAccessor)